zanata-war/src/main/java/org/zanata/ZanataInit.java

```diff
@@ -430,7 +431,8 @@ private void logBanner(VersionInfo ver, AppServerVersion appServerVersion) {
         }
         log.info("  AS version: " + appServerVersion.asVersion.orNull());
         log.info("  SCM: " + ver.getScmDescribe());
-        log.info("  Red Hat Inc 2008-2014");
+        log.info("  Red Hat Inc 2008-{}",
```

Comment:
I don't really care, but the issue was that the later year was wrong in *any* case. Having the current year *looks* good, as the human brain tends to see old years as bad.
"© year-year company" is a generally acceptable copyright label, and in a legal stoush having one is better than not having one.